.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
@noor-ahmed For the format, between this `[ xxx, yyy ]` and `[xxx, yyy]`.
Which one should we go for it?
For my idea, I'll go with the first option.
What do you think?

Reply:
for `{}` i prefer `{ xyz }`, for () and [] I prefer `[xyz, abc]`